hack/local-up-cluster.sh

```diff
@@ -316,6 +316,38 @@ function set_service_accounts {
     fi
 }
 
+function create_client_certkey {
```

Comment:
I'd like to be allowed to pass a CA through to here so I can create one for front-proxy authentication easily.